### DISCUSSION TITLE
Hero 13 Wide = 1

### DISCUSSION THREAD

**Discussion by DTL77** [gopro/labs#1124] [Q&A]
In Labs, Wide is not a setting for the H13. Does this mean the H13 has wide as default now?

**Comment by dnewman-gpsw** [comment]
*WIDE=1 is a setting on HERO12 and HERO13, and GP-Log defaults to a wide gamut on both.

**Comment by DTL77** [comment]
Thanks for the update.  I was wondering because the Hero 13 is not listed as being supported. Only the 11 and 12 are listed for Wide.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by dnewman-gpsw** [reply]
The page https://gopro.github.io/labs/control/extensions/ maybe just needs a refresh.

HERO13 has 105% of the features of HERO12 (i.e. all plus more.)

**Comment by DTL77** [comment]
Ah now it has it 🙂


David T. Long
Professor Emeritus & Adjunct Professor
Aqueous & Environmental Geochemistry
Dept. of Earth and Environmental Sciences
288 Farm Lane, Rm 207
Michigan State University
East Lansing, Michigan 48824
my office: 517.353.9618
dept office: 517.355.4626
email: ***@***.***

[cid:25439b0b-bbc0-4bac-9b19-167736080bf6]

Michigan State University occupies the ancestral, traditional, and contemporary Lands of the
      Anishinaabeg–Three Fires Confederacy of Ojibwe, Odawa, and Potawatomi peoples.
                   The University resides on Land ceded in the 1819 Treaty of Saginaw.

    Sometimes you win, sometimes you lose, sometimes it rains. (Bull Durham, 1988)
Spartan Strong
________________________________
